modules/data.atmosphere/R/met.process.R

```diff
@@ -531,7 +531,7 @@ browndog.met <- function(browndog, source, site, start_date, end_date, model, di
   userpass <- paste(browndog$username, browndog$password, sep = ":")
   curloptions <- list(userpwd = userpass, httpauth = 1L, followlocation = TRUE)
   result <- RCurl::postForm(paste0(browndog$url, formatname, "/"),
```

Comment:
I'm guessing this is intentionally incomplete for now and will use the same pattern you develop in `convert.input`, but here's a placeholder to remind us to finish it :)